testsuite/run_sets/testsuite.yml

```diff
@@ -111,6 +111,7 @@
 - features/srv_spacewalk_channel.feature
 - features/allcli_software_channels.feature
 - features/allcli_software_channels_dependencies.feature
+- features/srv_change_task_schedule.feature
```

Comment:
I hope this is the correct location now. I could not find any other "task related" feature, so I chose this location as this is changing the schedule of repo sync. Since the schedules are present from the very beginning I was under the impression that this test would be allowed to run at the very beginning. So far I did not yet understand the logic/system of test run order (eg. why is the test for rebooting all the clients performed before all the links on the mainpage are checked? To me the order looks rather random.). And as they are supposed to be idempotent I think the order should not matter at all. But in the end I really do not care.

Reply:
OK. I thought we had other scheduling-centered features and I did not check.

You are right. In theory, the test run order is supposed to be arbitrary and changed at will (idempotency).

In practice, we tried to shuffle the secondary features at random and it failed quite badly. This shows us that we still need to make some progress on the idempotency front.

Using a more or less logical order is just about making it easier to read the results. Also, sometimes, we want to run a few features and then press `Ctrl-C`, just to make sure that it more or less works. In such a context, it's better if the most important or most error-catching features are at the beginning. I do not think this new feature is of capital importance, so it can go somewhere into the middle.

Ping @0rnela @srbarrios @lkotek @aliouliaki for awareness.